File: Amazon.Lambda/Felipearpa.Tyche.Pool/src/Domain/PoolGamblerScore.fs
namespace Felipearpa.Tyche.Pool.Domain

open Felipearpa.Type

type PoolGamblerScore =
    { PoolId: Ulid
      PoolName: NonEmptyString100
      GamblerId: Ulid
      GamblerUsername: NonEmptyString100
      Position: int option
      BeforePosition: int option
      Score: int option
      BeforeScore: int option }

File: Android/data/pool/src/main/java/com/felipearpa/tyche/data/pool/domain/PoolGamblerScore.kt
package com.felipearpa.tyche.data.pool.domain

data class PoolGamblerScore(
    val poolId: String,
    val poolName: String,
    val gamblerId: String,
    val gamblerUsername: String,
    val position: Int?,
    val beforePosition: Int?,
    val score: Int?,
)
